src/components/input/select/index.tsx

```diff
@@ -35,11 +35,11 @@ export class SmoothlyInputSelect implements Input, Editable, Clearable, Componen
 	private itemHeight: number | undefined
 	@Element() element: HTMLSmoothlyInputSelectElement
 	@Prop() name = "selected"
-	@Prop() showArrow = true
 	@Prop({ reflect: true, mutable: true }) color?: Color
 	@Prop({ reflect: true, mutable: true }) looks: Looks = "plain"
 	@Prop({ reflect: true, mutable: true }) showSelected?: boolean = true
 	@Prop({ reflect: true, mutable: true }) readonly = false
+	@Prop() showArrow?: boolean
```

Comment:
make it "noArrow" and default to false, so you can set it using the good way when you use the component.

Reply:
I think this is better given that the default behaviour is to only show arrows when `looks == "border"`